persistence/sql/migrations/templates/20210913095309_identity_recovery_tokens.up.fizz

```diff
@@ -0,0 +1,2 @@
+add_column("identity_recovery_tokens", "identity_id", "uuid", {"size": 36})
```

Comment:
This should have a foreign key :)

Reply:
Fixed in https://github.com/ory/kratos/pull/1750/commits/c6f3d98a92cdbb928e105fa1ae653e5796c6a155